src/api/environment.cc

```diff
@@ -783,6 +814,9 @@ Maybe<void> InitializePrimordials(Local<Context> context) {
     return Nothing<void>();
   }
 
+  Local<Object> private_symbols =
+      InitializePrivateSymbols(context, isolate_data);
```

Comment:
`InitializePrivateSymbols` should not fail. But let's return a `MaybeLocal<Object>` in `InitializePrivateSymbols` and return early here, in case of errors.

```suggestion
  Local<Object> private_symbols;
  if (!InitializePrivateSymbols(context, isolate_data).ToLocal(&private_symbols)) {
    return Nothing<void>();
  }
```